rllib/algorithms/ppo/torch/ppo_torch_learner.py

```diff
@@ -39,16 +39,24 @@ def compute_loss_per_module(
         # learning rate for that agent.
         # TODO (Kourosh): come back to RNNs later
 
-        action_dist_class = self.module[module_id].unwrapped().action_dist_cls
-        curr_action_dist = action_dist_class.from_logits(
+        action_dist_class_train = (
+            self.module[module_id].unwrapped().get_train_action_dist_cls()
```

Comment:
nit: you don't need to call unwrap() as they are DDPTorchRLModules.

Reply:
I do. Modules are not always DDPTorchRLModules, but sometimes also `DDPTorchRLModuleWithTargetNets` (APPO). ;)

Reply:
Oh why is that one not implementing the get_train_action_dist_cls()? it should imo. All RLModules should implement these new APIs.